### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by TheBusyBot**

### CODE REVIEW

File: src/main/resources/languages/lv/messages.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ guide:
     languages: 'Izvēlies savu vēlamo valodu,'
     credits: 'Slimefun4 Ieguldītāji'
   credits:
-    commit: "Ieguldījums"
+    commit: "\\\\t\nIeguldījums"
```
Comment (WalshyDev) [inline src/main/resources/languages/lv/messages.yml:38]: what's going on here :thinking:

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: src/main/resources/languages/lv/messages.yml (hunk 1 above)
Comment (TheBusyBiscuit) [reply src/main/resources/languages/lv/messages.yml]: gitlocalize had a lot of issues with tab and new line characters.
Luckily, Crowdin detects them, so we we can fix it

**Review by TheBusyBiscuit — APPROVED** [review]

**Comment by TheBusyBiscuit** [comment]
Damn, I forgot that I wanted to squash crowdin PRs...

**Comment by TheBusyBiscuit** [comment]
Can honestly auto merge and squash these upon approval 🤔

**Pull request merged**